entry_timeseries.go

```diff
@@ -83,6 +91,33 @@ func (entry TimeseriesEntry) Create() error {
 	return makeErrorOrNil(err)
 }
 
+// InsertColumns : insert columns in a existing timeseries
+func (entry *TimeseriesEntry) InsertColumns(cols ...TsColumnInfo) error {
+	colsToInsert := TsColumnInfos(cols)
```

Comment:
It may be purely personal, but I find `TsColumnInfos` (with 's') more confusing than clear.
I'll see the use of `[]TsColumnInfo` (without 's') clearer to the reader.